### DISCUSSION TITLE
TP3 - Fusionadora de arreglo

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#143] [Preguntas y respuestas] [answered]
Hola @dteira , te consultamos con @agosgit porque entendemos que la fusionadora de arreglos que hicimos es general, pero no entendemos como hacer para que pasen los parámetros de cada caso en particular, para el primer arreglo y para el segundo arreglo porque tienen definidas variables diferentes por ser arreglos diferentes, no se si me explico.

Esperamos tu respuesta, 
Graciaaas
Agos y Daysi

![borrador-fusionadora](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/ca1afe75-3f83-451a-a075-c8df80112b2b)

**Comment by martinvilu** [comment]
Lo único que le falta a la copiadora para ser genérica (y que la puedas usar en este ejercicio), es una posición de destino, aunque es posible resolverlo sin este argumento, su uso es mas complejo ya que requiere de aritmetica de punteros.

**Reply by martinvilu** [reply]
Ya que así, ambas copian en destino, a partir de la posición 0.

Usá arreglos así para verlo facilmente

```C
int arreglo_uno_[] = {1, 2, 3, 4, 5};
int arreglo_dos[] = {10, 20, 30, 40, 50};
```

Por otro lado, este ejercicio permite el uso de las funciones para cargar y mostrar arreglos del TP3.

**Reply by Clirussa** [reply]
Quiero saber si entendí bien ¿Para hacerla genérica solo con punteros podría hacerla sino si le tengo que poner los argumentos que deben ingresar?

**Reply by martinvilu** [reply, marked as answer]
> Para hacerla genérica solo con punteros podría hacerla 

Sí
> si le tengo que poner los argumentos que deben ingresar

Esto sería, a donde empezar a 'pegar' lo que estas 'copiando'
